drawing/pdf_draw_shapes.go

```diff
@@ -187,16 +187,16 @@ func drawPolyBezierCurve(c *creator.Creator) error {
 		draw.NewCubicBezierCurve(450, 300, 446, 288, 442, 260, 450, 250), // leaf
 	})
 
-	curve.SetBorderColor(creator.ColorBlue)
-	curve.SetFillColor(creator.ColorGreen)
+	curve.SetBorderColor(creator.ColorCMYKFromArithmetic(1.0, 1.0, 0.0, 0.0))
```

Comment:
maybe not the right place?  Just need one example to show CMYK colors.  For people specifically looking for that.

Reply:
@3ace let's leave cmyk colors out of this example and put that into the specific _cmyk example.  Can keep this color like it was